tests/contenttypes_tests/test_models.py

```diff
@@ -2,14 +2,20 @@
 from django.contrib.contenttypes.models import ContentType, ContentTypeManager
 from django.contrib.contenttypes.prefetch import GenericPrefetch
 from django.db import models
-from django.db.migrations.state import ProjectState
-from django.test import TestCase, override_settings
+from django.db.migrations.state import ModelState, ProjectState
+from django.test import TestCase, TransactionTestCase, override_settings
 from django.test.utils import isolate_apps
 
 from .models import Author, ConcreteModel, FooWithUrl, ProxyModel
 
 
-class ContentTypesTests(TestCase):
+class ContentTypesTests(TransactionTestCase):
```

Comment:
I'm surprised `TransactionTestCase` is required here, what's the reason for it? It makes the test way slower as it cannot use transactions to isolate database changes and must flush untouched tables.

Reply:
There was a test failure in the CI and I panicked 😁 I was missing the `@isolate_apps("contenttypes_tests")` 
I will revert 👍